src/pfe/file-watcher/server/package.json

```diff
@@ -12,8 +12,8 @@
     "tslint": "tslint -c tslint.json -p tsconfig.json",
     "eslint": "eslint .",
     "test": "npm run unit:test && npm run functional:test",
-    "unit:test": "NODE_ENV=test CW_LOCALES_DIR=$(pwd)/src/utils/locales/ CW_WORKSPACE=$(pwd) nyc mocha test/unit-test/unit.test.ts",
-    "unit:test:xml": "NODE_ENV=test CW_LOCALES_DIR=$(pwd)/src/utils/locales/  CW_WORKSPACE=$(pwd) nyc mocha test/unit-test/unit.test.ts --colors --reporter mocha-jenkins-reporter",
+    "unit:test": "NODE_ENV=test CW_LOCALES_DIR=$(pwd)/src/utils/locales/ CW_WORKSPACE=$(pwd) CW_PROJECTDATA_DIR=$(pwd)/test/resources/ nyc mocha test/unit-test/unit.test.ts",
```

Comment:
Since we have defined the `CW_WORKSPACE` to be `$(pwd)`, can you simply define `CW_PROJECTDATA_DIR` in the app.config.js ? to be `path.join(CW_WORKSPACE, "/test/resources/")`?